### DISCUSSION TITLE
Does macad-gym support multi-agent algos training?

### DISCUSSION THREAD

**Discussion by dineshresearch** [praveen-palanisamy/macad-gym#61] [General]
will this currently support multi agent RL(MARL) algos like https://github.com/decisionforce/CoPO

I am particularly interested in the Multi-vehicle cooperation and competition at intersection case?

any help would be highly appreciated

**Comment by praveen-palanisamy** [comment]
Hi @dineshresearch ,
Yes, you can use a multi-agent RL algorithm like the Coordinated Policy Optimization (CoPO). Please see the [MACAD-Agents](https://github.com/praveen-palanisamy/macad-agents) repository for multi-agent training code examples.

**Comment by praveen-palanisamy** [comment]
Converting this question to a GitHub discussion. Please mark the discussion as answered if you got the answer to your question.

**Comment by 6Lackiu** [comment]
> will this currently support multi agent RL(MARL) algos like https://github.com/decisionforce/CoPO
> 
> I am particularly interested in the Multi-vehicle cooperation and competition at intersection case?
> 
> any help would be highly appreciated

@dineshresearch Hi, have you successfully run CoPO in macad-gym?
